<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadBackupWizardPageSelectBackupType</class>
 <widget class="QWizardPage" name="LoadBackupWizardPageSelectBackupType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Backup Type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblHeader">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Please select the type of backup you want to load:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbtnIdBackup">
     <property name="text">
      <string>ID Backup</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbtnDataBackup">
     <property name="text">
      <string>Data Backup</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbtnLegacyClientConfiguration">
     <property name="text">
      <string>OpenMittsu 0.9.2 client configuration (legacy)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblExplanation">
     <property name="text">
      <string>An ID Backup comes in the form of a text-string that looks like this: HTTH-C2A1-O1Q2-.... It contains only your private key and ID - no contacts or messages.
A data backup on the other hand contains your private key, your ID plus all contacts, groups and messages. A data backup is exported by Threema as a password-protected *.zip file. If you chose to also included media files in your data backup, these can be imported as well.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
